### PR TITLE
Upgrade dataloader-sequelize to v2

### PR DESCRIPTION
**Pull request by gmarcos87**
resolve https://github.com/opencollective/opencollective/issues/2094

### CODE REVIEW

File: server/constants/currencies.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  EUR: { fxrate: 0.83, format: value => `€${value}` },
+  GBP: { fxrate: 0.74, format: value => `£${value}` },
+  INR: { fxrate: 64.29, format: value => `₹${value}` },
+  MXN: { fxrate: 17.74, format: value => `${value} MXN` },
+  SEK: { fxrate: 7.96, format: value => `kr ${value}` },
+  USD: { fxrate: 1, format: value => `$${value}` },
+  UYU: { fxrate: 29.17, format: value => `$U ${value}` },
+};
+
+export default fxRates;
```
Comment (znarf) [inline server/constants/currencies.ts]: Not sure what happened, but this should not be there I think.

File: server/graphql/v1/mutations/members.js [hunk 2]
```diff
@@ -1,6 +1,99 @@
...
+export async function editMembership(_, args, req) {
+  const member = await models.Member.findByPk(args.id);
+
+  // Only admin of member collective can edit the membership
+  if (!member || !req.remoteUser || !req.remoteUser.isAdmin(member.MemberCollectiveId)) {
+    throw new errors.Unauthorized("This member doesn't exist or you don't have the permission to edit it");
+  }
+
+  return member.update(pick(args, ['publicMessage']));
+}
```
Comment (znarf) [inline server/graphql/v1/mutations/members.js]: Looks also unrelated.

File: server/constants/currencies.ts (hunk 1 above)
Comment (gmarcos87) [reply server/constants/currencies.ts]: that when @CanerSezgin  published his branch he sent some undeceived changes. I'm going to do some cleaning.

**Comment by gmarcos87** [comment]
I did some cleaning and updated via rebase instead of merge with master.

**Comment by CanerSezgin** [comment]
I couldn't test it out but I've taken a look at the files changes after **gmarcos87**'s last update. it seems fine to me. 

When I worked on this, there was some errors at the point that I left and I had tried to change/update tests as well like @brymut tried. (It had contained really major changes)
However if you are saying this is working now well done, great work @gmarcos87 

I guess the fixes on this issue https://github.com/opencollective/opencollective/issues/2607 sorted out this one as well

**Comment by znarf** [comment]
Triggering the E2E tests because the changes are massive. Let's see how it goes https://github.com/opencollective/opencollective-frontend/pull/3107

**Comment by znarf** [comment]
One E2E test failed https://circleci.com/gh/opencollective/opencollective-frontend/60966, re-triggered it to see if it fails again.

**Comment by znarf** [comment]
Looks like the E2E are passing!
